packages/app/client/src/ui/pages/Account/components/Notifications.js

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from 'react'
+import Typography from '@material-ui/core/Typography'
+import { Switch } from '@material-ui/core'
+import NotificationsActivity from './NotificationsActivity'
+import { makeStyles } from '@material-ui/core/styles'
+
+const useStyles = makeStyles(() => ({
+  wraper: {
+    maxWidth: '470px'
+  },
+  root: {
+    width: 54,
+    height: 24,
+    padding: 0,
+    margin: '3% 3% 3% 0'
+  },
+  switchBase: {
+    padding: 1,
+    '&$checked': {
+      transform: 'translateX(30px)',
+      color: 'white',
+      '& + $track': {
+        backgroundColor: '#15B54A',
+        opacity: 1,
+        border: 'none'
+      }
+    },
+    '&$focusVisible $thumb': {
+      color: '#15B54A',
+      border: '6px solid #fff'
+    }
+  },
```

Comment:
Is it the same style applied in NotificationsActivity ? I think it will be a good idea make it shared between this two components